testing/python/fixtures.py

```diff
@@ -1435,6 +1435,19 @@ def test_two(self):
         reprec = pytester.inline_run()
         reprec.assertoutcome(passed=2)
 
+    def test_empty_usefixtures_marker(self, pytester: Pytester) -> None:
+        pytester.makepyfile(
+            """
+            import pytest
+
+            @pytest.mark.usefixtures()
+            def test_one():
+                assert 1 == 1
+        """
+        )
+        reprec = pytester.inline_run()
+        reprec.assertoutcome(failed=1)
```

Comment:
This is good, but I feel we should test the actual message, using `pytest.runpytest()` and `fnmatch_lines()`.